h2/src/main/org/h2/store/fs/encrypt/FileEncrypt.java

```diff
@@ -65,6 +66,8 @@ public class FileEncrypt extends FileBaseDefault {
 
     private byte[] encryptionKey;
 
+    private FileEncrypt source;
```

Comment:
I think this field should be set to `null` after initialization.

Reply:
Sure